sysbuild/Kconfig.wifi

```diff
@@ -4,32 +4,32 @@
 
 menu "Wi-Fi"
 
-menuconfig WIFI_NRF700X
+menuconfig WIFI_NRF70
 	bool "nRF700x"
 
-if WIFI_NRF700X
+if WIFI_NRF70
 
-choice WIFI_NRF700X_OPER_MODES
+choice WIFI_NRF70_OPER_MODES
 	bool "nRF700x operating mode"
-	default WIFI_NRF700X_SYSTEM_MODE
+	default WIFI_NRF70_SYSTEM_MODE
 	help
 	  Select the operating mode of the nRF700x driver
 
-config WIFI_NRF700X_SYSTEM_MODE
-	bool "Enable nRF700X system mode"
+config WIFI_NRF70_SYSTEM_MODE
+	bool "Enable nRF70 system mode"
```

Comment:
Shouldn't the text remain as nRF700x? Or nRF70x? Since this doesn't apply to just a single product called nRF70 but the series

Reply:
`the nRF70 Series` is appropriate.

Reply:
I will do this in a separate PR upstream and then pull to NCS.